### PR TITLE
Added some resources

### PR DESCRIPTION
**Pull request by virtualwiz1**
## What does this PR do?
Add resource(s)

## For resources
Android, MySQL, Raspberry-Pi, Linux
### Description
Added new resources for the mentioned titles

### Why is this valuable (or not)?
Mentioned Links provide more extensive information on the written topics

### How do we know it's really free?
I have tested them as well as use it for myself (no need to register or create account)

### For book lists, is it a book? For course lists, is it a course? etc.
They cover certain topic and project guides

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -264,6 +264,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Google Android Developer Training](https://developer.android.com/guide)
 * [Styling Android](https://blog.stylingandroid.com)
 * [The Busy Coder's Guide to Android Development](https://commonsware.com/Android/4-2-free) (PDF - older versions)
+* [Android Open source Projects](https://source.android.com) - Open source projects on android to learn from
```
Comment (metabiswadeep) [inline books/free-programming-books-langs.md:267]: Move to line 258

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1425,6 +1426,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [UNIX Systems Programming for SVR4](http://www.bitsinthewind.com/about-dac/publications/unix-systems-programming) - David A. Curry
 * [Upstart Intro, Cookbook and Best Practises](http://upstart.ubuntu.com/cookbook/)
 * [What Every Programmer Should Know About Memory](http://www.akkadia.org/drepper/cpumemory.pdf) (PDF)
+* [Learning Linux with Termux on Android](https://medium.com/@virtualwiz/hacking-with-android-termux-581bccae9ae2) - Guide to using termux
```
Comment (metabiswadeep) [inline books/free-programming-books-langs.md:1429]: Move to line 1404

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -2086,6 +2089,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Raspberry Pi Users Guide (2012)](http://www.cs.unca.edu/~bruce/Fall14/360/RPiUsersGuide.pdf) - Eben Upton (PDF)
 * [The Official Raspberry Pi Handbook 2023](https://magpi.raspberrypi.com/books/handbook-2023) - The MagPi magazine (PDF)
 * [The Official Raspberry Pi Project Book 1 (2015)](https://magpi.raspberrypi.com/books/projects-1) (PDF)
+* [Guide to making suspicious activity tracking AI camera](https://www.electronicsforu.com/electronics-projects/raspberry-pi-diys/suspicious-activity-tracking-ai-camera) - Extensive HTML Doc
```
Comment (metabiswadeep) [inline books/free-programming-books-langs.md:2092]: Move to line 2087

**Review by metabiswadeep — CHANGES_REQUESTED** [review]

**Pull request closed without merging**